operator-metadata/image.yaml

```diff
@@ -0,0 +1,47 @@
+schema_version: 1
+
+name: amq7/amq-streams-operator-metadata
+description: "AMQ Streams image containing the operator metadata"
+version: "1.3.0"
+from: ubi7-minimal:7-released
```

Comment:
There was mention of using "scratch" as the base image for these operator-metadata images but I am not sure if it has been released yet

Reply:
we have to align to the new 1.4 version for all the other files but I would do it in a different PR